package.json

```diff
@@ -2,6 +2,7 @@
   "name": "@aligent/serverless-aws-nodejs-service-template",
   "version": "2.0.0",
   "description": "Aligent AWS microservices template using Typescript and Serverless Framework",
+  "packageManager": "pnpm@9.12.1",
```

Comment:
I think we need a brief discussion here over if we should use [Corepack](https://github.com/nodejs/corepack) or not. As far as I understand, this `packageManager` property is supported by Corepack only.
 Nodejs recently voted to not enabling Corepack & may [remove it from Nodejs distribution in the future](https://socket.dev/blog/node-js-takes-steps-towards-removing-corepack).
 For now, I think we should stick with the plain old `engines`. Eg:
```json
"engines": {
        "node": ">=20",
        "pnpm": ">=9"
    }
```
- https://www.marcusoft.net/2015/03/packagejson-and-engines-and-enginestrict.html
- https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines
- https://pnpm.io/package_json

Reply:
Fixed this. It meant that I needed to specify the version on the github actions package yaml since it the pnpm build relies on that "packageManager" specification otherwise.